test/unit/voter-proxy.js

```diff
@@ -263,7 +263,7 @@ describe('unit - VoterProxy', function () {
     await bal.mint(voterProxy.address, ONE_HUNDRED_ETHER);
     expect(await bal.balanceOf(voterProxy.address)).to.equals(ONE_HUNDRED_ETHER);
     // distro mock does nothing
-    await voterProxy.connect(anotherUser).claimFees(distro.address, bal.address);
+    await voterProxy.connect(anotherUser).claimFees(distro.address, [bal.address]);
```

Comment:
Can we check claimFees for a couple of tokens?

Reply:
This just for voterProxy test functionality. In controller.js line 159, we are testing the claiming of `incentiveRewardToken`